.circleci/config.yml

```diff
@@ -90,7 +90,7 @@ commands:
 jobs:
   build_check:
     docker:
-      - image: aeternity/infrastructure:master
+      - image: aeternity/infrastructure:stable
```

Comment:
Not sure if I asked already, but what's the purpose of this ? You can't integrate BC breaking features if this is set to stable, because you need to run at least the latest master.

Reply:
Consider this as typo ;) will fix

Reply:
Consider this as typo ;) will fix